### PR TITLE
Feature/minor changes

### PR DESCRIPTION
**Pull request by Drish-xD**
# TODO
- ADD `broadcast` in session_types
- ADD `Code` in AuthType
- Handle tnschools options using tnstudents
- Redirection as true by default
- UI chanege 
  - Name field to basic step
  - Move platform to above
  - Table reorder

### CODE REVIEW

File: src/app/(home)/Table/Filters.tsx [hunk 1]
```diff
@@ -3,43 +3,136 @@ import {
...
+      const authGroupId = apiOptions.group?.find((item) => item.value === groupFilter)?.id;
+      const filteredQuizBatchOptions =
+        apiOptions?.batch?.filter((item) => item.groupId === authGroupId && !item.parentId) ?? [];
+      fields.parentId.options = filteredQuizBatchOptions;
+      fields.parentId.show = true;
+
+      if (parentIdFilter) {
+        const quizBatchId = apiOptions.batch?.find((item) => item.value === parentIdFilter)?.id;
+        const filteredClassBatchOptions =
+          apiOptions?.batch?.filter((item) => item.parentId === quizBatchId) ?? [];
```
Comment (suryabulusu) [inline src/app/(home)/Table/Filters.tsx:73]: how come classBatchId not being used

File: src/app/session/[type]/Steps/helper.ts [hunk 2]
```diff
@@ -135,11 +137,15 @@ export const setParentBatchOptions = (
   fieldsSchema: FieldSchema<basicFields>
 ) => {
   if (!value) return;
-
   const authGroupId = apiOptions.group?.find((item) => item.value === value)?.id;
-  const filteredQuizBatchOptions = apiOptions?.batch?.filter(
-    (item) => item.groupId === authGroupId && !item.parentId
-  );
+  let filteredQuizBatchOptions: ExtendedOptions[] = [];
+  if (authGroupId === 14) {
```
Comment (suryabulusu) [inline src/app/session/[type]/Steps/helper.ts:142]: can u comment this is for tnschools

Comment (suryabulusu) [reply src/app/session/[type]/Steps/helper.ts:142]: infact we dont know if authgroupid will always be 14.. maybe a better way is to GET authgroupid given auth group name

**Review by suryabulusu — COMMENTED** [review]
left comments.. looking good otherwise

File: src/app/(home)/Table/Filters.tsx (hunk 1 above)
Comment (Drish-xD) [reply src/app/(home)/Table/Filters.tsx]: Here I'm setting quiz batch options for filters using authGroup. 
and class batch options using the selected quizBatch

Their is no filter which depends on the classBatchId, that's why its not being used.

File: src/app/session/[type]/Steps/Platform/Quiz.tsx [hunk 3]
```diff
@@ -53,11 +53,6 @@ const QuizForm: FC = () => {
         placeholder: 'Select a test purpose',
         label: 'Test Purpose',
       },
-      name: {
```
[the remark this answers is not in the document]
Comment (Drish-xD) [reply src/app/session/[type]/Steps/Platform/Quiz.tsx:56]: It's being shifter to the Basic Step, so deleted from platform step (Quiz or Live).

File: src/app/session/[type]/Steps/helper.ts (hunk 2 above)
Comment (Drish-xD) [reply src/app/session/[type]/Steps/helper.ts:142]: Should I Handle this check using Name?

Comment (Drish-xD) [reply src/app/session/[type]/Steps/helper.ts]: Done

File: src/app/session/[type]/Steps/Platform/Quiz.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Drish-xD) [reply src/app/session/[type]/Steps/Platform/Quiz.tsx:56]: Session Name

**Review by suryabulusu — APPROVED** [review]

**Pull request merged**